common/src/WooCommerce.php

```diff
@@ -62,12 +62,35 @@ public function autoSyncNotice() {
     public function set_order_language($order_id) {
         /** @var WC_Order $order */
         $order = wc_get_order($order_id);
-        if (!$order->get_meta('wpml_language') && defined('ICL_LANGUAGE_CODE')) {
-            $order->update_meta_data('wpml_language', ICL_LANGUAGE_CODE);
+        $language_code = $this->detectOrderLanguage();
+        if (!$order->get_meta('wpml_language') && $language_code) {
+            $order->update_meta_data('wpml_language', $language_code);
             $order->save_meta_data();
         }
     }
 
+    private function detectOrderLanguage() {
+        if (defined('ICL_LANGUAGE_CODE')) { // WPML
+            return ICL_LANGUAGE_CODE;
+        } elseif (isset($_SERVER['HTTP_CLONABLE_TARGET_LANGUAGE'])) { // Clonable
+            return sanitize_text_field($_SERVER['HTTP_CLONABLE_TARGET_LANGUAGE']);
```

Comment:
For `if` statements with `return` (ie, guard clauses), there's no need to use `else`:

```suggestion
        }
        if (isset($_SERVER['HTTP_CLONABLE_TARGET_LANGUAGE'])) { // Clonable
            return sanitize_text_field($_SERVER['HTTP_CLONABLE_TARGET_LANGUAGE']);
```